p2p/peer.go

```diff
@@ -46,6 +46,7 @@ type Peer interface {
 	HasChannel(chID byte) bool // Does the peer implement this channel?
 	Send(e Envelope) bool      // Send a message to the peer, blocking version
 	TrySend(e Envelope) bool   // Send a message to the peer, non-blocking version
+	TrySendMarshalled(e MarshalledEnvelope) bool
```

Comment:
Is anyone outside the p2p package using this method?

If not, there is no point in adding it to the public interface.